ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -2538,6 +2538,29 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-ipi-sno
+- as: gcp-ipi-workers-marketplace-f14
+  cron: 13 20 12,27 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      COMPUTE_OSIMAGE: projects/redhat-marketplace-public/global/images/redhat-coreos-ocp-413-x86-64-202305021736
+      E2E_RUN_TAGS: '@amd64 and @gcp-ipi and @network-ovnkubernetes'
+      TAG_VERSION: '@4.14'
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-gcp-ipi-workers-marketplace
+- as: gcp-ipi-workers-marketplace-f28-destructive
+  cron: 22 11 9 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      COMPUTE_OSIMAGE: projects/redhat-marketplace-public/global/images/redhat-coreos-ocp-413-x86-64-202305021736
```

Comment:
4.14 test using 4.13 image ?

Reply:
@liangxia yes, it's expected. That's the only one Marketplace Image available for 4.13+

Reply:
Ack

Reply:
Yes, there are 2 GCP marketplace RHCOS images, one for 4.12 & earlier, another for 4.13 & later, see the [KCS](https://access.redhat.com/articles/6976804). Thanks!